README.md

```diff
@@ -15,18 +15,22 @@ Installing the dbt-dremio package will install or update dbt-core to version 1.3
 > Prior to version 1.1.0b, dbt-dremio was created and maintained by [Fabrice Etanchaud](https://github.com/fabrice-etanchaud) on [their GitHub repo](https://github.com/fabrice-etanchaud/dbt-dremio). Code for using Dremio REST APIs was originally authored by [Ryan Murray](https://github.com/rymurr). Contributors in this repo are credited for laying the groundwork and maintaining the adapter till version 1.0.6.5. The dbt-dremio adapter is maintained and distributed by Dremio starting with version 1.1.0b.
 
 ## Getting started
-
+---
 - [Install dbt](https://docs.getdbt.com/docs/installation)
    - Note that dbt-dremio requires dbt-core 1.3.0.
 - Read the [introduction](https://docs.getdbt.com/docs/introduction/) and [viewpoint](https://docs.getdbt.com/docs/about/viewpoint/)
 
 ## Join the dbt Community
-
+---
 - Be part of the conversation in the [dbt Community Slack](http://community.getdbt.com/)
 - Read more on the [dbt Community Discourse](https://discourse.getdbt.com)
 
 ## Reporting bugs and contributing code
-
-- Want to report a bug or request a feature? Let us know by opening [an issue](https://github.com/dremio/dbt-dremio/issues/new)
-
-
+---
+- Open bugs and feature requests can be found at [dbt-dremio's GitHub issues](https://github.com/dremio/dbt-dremio/issues).
+- Want to report a bug or request a feature? Let us know by on [Slack]([Slack](http://community.getdbt.com/)), or opening [an issue](https://github.com/dremio/dbt-dremio/issues/new)
```

Comment:
We do have a link to the slack channel now https://getdbt.slack.com/archives/C049G61TKBK

Reply:
I followed the example from Snowflake where it just went to the general dbt community Slack, but can add the link to ours.